### PR TITLE
add missing i18n key for closed OC

### LINKED ISSUE #5507: Missing translation key - Closed OC info
## Description

The info appearing for closed OCs is missing from the translation key file en.yml:
"The order cycle you've selected has just closed. Please try again!"

## Steps to Reproduce

1. Customer - add items to the cart, and proceed to checkout
2. Admin - Close the OC from which the customer has the items from
3. Customer - While on checkout, click Shops/Boutiques (below):

image

### PR DESCRIPTION
**Pull request by luchiago**
#### What? Why?

Closes #5507

Added one missing key for warning about closed OC

#### What should we test?

Close the OC while exists a customer with products from the selected OC on checkout

#### Release notes

Changelog Category: Fixed

### CODE REVIEW

**Comment by sauloperez** [comment]
ping @luisramos0

File: app/controllers/application_controller.rb [hunk 1]
```diff
@@ -93,8 +93,7 @@ def check_hub_ready_for_checkout
     if current_distributor_closed?
       current_order.empty!
       current_order.set_distribution! nil, nil
-      flash[:info] = "The hub you have selected is temporarily closed for orders. "\
-        "Please try again later."
+      flash[:info] = I18n.t('check_hub_ready_for_checkout')
```
Comment (luisramos0) [inline app/controllers/application_controller.rb:96]: this key is not well named. the key should describe the text, not the method where it appears.
maybe 'order_cycles_closed_for_hub' ?

Comment (luchiago) [reply app/controllers/application_controller.rb:96]: Yes, it's better. I will change.

**Comment by filipefurtad0** [comment]
Hey @luchiago ,

The warnings render nicely and the translations you added should appear correctly in production:

image

image

Thank you! :+1:

**Pull request merged**